kernel/kernel-config-server/src/main/java/io/mosip/kernel/config/server/controller/RefreshController.java

```diff
@@ -37,7 +37,7 @@ public class RefreshController {
 
     @GetMapping("/refresh")
     public Map<String, String> refreshContext(@RequestParam("servicename") String serviceName) {
-        logger.info("refreshContext invoked with serviceName : {}", serviceName);
+        logger.info("refreshContext invoked with serviceName : {}", serviceName.replaceAll("[\n\r\t]", "_"));
```

Comment:
Lets call it out as sanitized serviceName in the logs to avoid confustions in the future.

Reply:
Done, modified the log line.